packages/modal/components/DialogModalWithFooter.tsx

```diff
@@ -22,9 +23,11 @@ class DialogModalWithFooter extends React.PureComponent<
       <DialogModal
         footerContent={
           <div className={flex({ align: "center", justify: "space-between" })}>
-            <div className={flexItem("shrink")}>
-              <SecondaryButton onClick={onClose}>{closeText}</SecondaryButton>
-            </div>
+            {closeText !== "" || closeText != null ? (
```

Comment:
Non-blocking nitpick: wouldn't `closeText && ()` work just as well because an empty string is falsy?